incubator/gerrit/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
+name: gerrit
+version: 0.1.2
```

Comment:
Since this is a new chart, the version should start at `0.1.0`.

Reply:
Noted, I'll make the change.